software/drivers/sd8887/src/bluetooth/bt/bt_proc.c

```diff
@@ -631,13 +648,22 @@ bt_histogram_proc_open(struct inode *inode, struct file *file)
 }
 
 /** Histogram proc fops */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
+static const struct proc_ops histogram_proc_fops = {
```

Comment:
And this changes are required since v5.6 because of https://github.com/torvalds/linux/commit/d56c0d45f0e27f814e87a1676b6bdccccbc252e9 I guess? If so, let's add that info in the commit log.